_posts/2022-08-08-async-processing.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Fang: Async background processing for rust
```

Comment:
Maybe it is better `Fang , Async ...` , also for some reason it is failing in hugo yaml

Reply:
done